exotica/Vector q0
string xml_file_
moveit_msgs/PlanningScene scene_
string group_name_
float32 max_time_
string problem_
string solver_
---
float32 planning_time_
exotica/Matrix solution_
bool succeeded_
---
float32 solving_time_
